### PR TITLE
131 include concerts and concertcontacts in sampledatautil

### LINKED ISSUE #131: Include Concerts and ConcertContacts in SampleDataUtil
When the app is first run, sample data for Persons are loaded, but not Concerts and ConcertContacts.

### PR DESCRIPTION
**Pull request by hj235**
Fixes #131

### CODE REVIEW

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -54,8 +124,8 @@ public static ReadOnlyAddressBook getSampleAddressBook() {
      */
     public static Set<Tag> getTagSet(String... strings) {
         return Arrays.stream(strings)
-                .map(Tag::new)
-                .collect(Collectors.toSet());
+            .map(Tag::new)
```
Comment (bbbbcccd) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java]: Check coding style if this is the right indentation. If it's correct, then leave it.

**Review by bbbbcccd — COMMENTED** [review]
lgtm, just check the indentation to be sure. Can merge after checking

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 1 above)
Comment (hj235) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java]: image

Good catch! pls review again

**Review by bbbbcccd — APPROVED** [review]
lgtm!

**Pull request merged**